setup.py

```diff
@@ -36,6 +36,10 @@ def requirements_for(version=None):
 requirements_version, links_version = requirements_for(version_info.major)
 install_requires = requirements_general | requirements_version
 dependency_links = links_general | links_version
+if version_info.major >=3:
```

Comment:
This is a great change. Thanks for the contribution. Do you think it would be useful to swap this block with [io.open](https://docs.python.org/2/library/io.html#io.open) so that we can use the same call to read the file in Python 2 and 3 without the need for the if-conditional?

Reply:
Yes, good idea -- will resubmit this weekend.

Reply:
Given that there hasn't been activity on this pull request in a while, I've made the change in 8f6b34c.